cms/grading/tasktypes/Batch.py

```diff
@@ -262,6 +265,7 @@ def evaluate(self, job, file_cacher):
             commands,
             job.time_limit,
             job.memory_limit,
+            max_processes,
```

Comment:
Yeah, and it's somewhat the reason I normally use a ternary; they're almost the same as an idiom, except cpython seems to have butchered the inbuilt. Whatever, they're both completely fine.